### PR TITLE
Update README.md with deprecation warning 

### PR DESCRIPTION
**Pull request by logankilpatrick**

### CODE REVIEW

**Comment by logankilpatrick** [comment]
Closes #15 and #18

**Comment by crstnbr** [comment]
@MikeInnes Could you please merge this? 

Issues came up [here](https://discourse.julialang.org/t/is-there-an-interface-to-mathematica-through-wolframescript/35727).

File: README.md [hunk 1]
```diff
@@ -1,5 +1,7 @@
 # Mathematica.jl
 
+## Warning: Mathematica.jl is no longer activly maintained. Head over to [MathLink.jl](https://github.com/JuliaInterop/MathLink.jl) for similar functionality.
```
Comment (MikeInnes) [inline README.md]:
```suggestion
## Warning: Mathematica.jl is no longer actively maintained. Head over to [MathLink.jl](https://github.com/JuliaInterop/MathLink.jl) for similar functionality. 
```

**Comment by logankilpatrick** [comment]
Good catch!

**Pull request merged**